### PR TITLE
Add EEx depracation to Deprecations.md

### PR DESCRIPTION
**Pull request by wojtekmach**
https://github.com/elixir-lang/elixir/pull/6258#discussion_r124097551

### CODE REVIEW

File: lib/elixir/pages/Deprecations.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ Deprecated feature                               | Deprecated in | Replaced by (
 :----------------------------------------------- | :------------ | :----------------------------
 `%{key => value}` in typespecs                   | [v1.6]        | `%{required(key) => value}` or `%{optional(key) => value}` (v1.4 and OTP 19)
 `Enum.partition/2`                               | [v1.6]        | `Enum.split_with/2` (v1.4)
+`<%=` in middle and end expressions              | [v1.5]        | `<%=` in the start expression (v1.0)
```
Comment (eksperimental) [inline lib/elixir/pages/Deprecations.md]: We should mention it belongs to EEx, and it should be placed at the end of the v1.5 deprecations list.

Comment (eksperimental) [inline lib/elixir/pages/Deprecations.md]: the replacement should be `<%` instead?

Comment (josevalim) [inline lib/elixir/pages/Deprecations.md]: Agreed. Maybe something like:

> Use `<%` (`=` is allowed only on start expressions)

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**